pmd-ui/src/main/resources/net/sourceforge/pmd/util/fxdesigner/fxml/designer.fxml

```diff
@@ -42,6 +44,12 @@
                                         <MenuItem fx:id="licenseMenuItem" mnemonicParsing="false" text="License" />
                                     </items>
                                 </Menu>
+                                <Menu mnemonicParsing="false" text="PMD Configuration">
+                                    <items>
+                                        <MenuItem fx:id="pmdconfig" mnemonicParsing="false" text="AuxClassPath SetUp"/>
```

Comment:
The naming here is sloppy too.

> text="AuxClassPath SetUp"

* The capitalisation is wrong
* All other menu items are named with the "verb + object" convention, eg "Open file" or "Export rule". This menu item should follow the same convention, e.g. be named "Set up the auxclasspath".

> fx:id="pmdconfig"

You're using the name of the *Menu* as the id of the Menu*Item*. FX ids also have a naming convention, that is, they should be *camelCase*, and be suffixed with their type. You can see eg `licenseMenuItem` a few lines above. Rename that to `setupAuxclasspathMenuItem`


Conventions are important, and even though they're usually implicit, you can usually pick them up just by looking around